discord/ui/container.py

```diff
@@ -0,0 +1,386 @@
+            The items contained in this section, up to 3.
+            Currently only supports :class:`~discord.ui.TextDisplay`.
+        accessory: Optional[:class:`Item`]
+            The section's accessory. This is displayed in the top right of the section.
+            Currently only supports :class:`~discord.ui.Button` and :class:`~discord.ui.Thumbnail`.
+        id: Optional[:class:`int`]
+            The section's ID.
+        """
+        # accept raw strings?
+
+        section = Section(*items, accessory=accessory, id=id)
+
+        return self.add_item(section)
+
+    def add_text(self, content: str, id: int | None = None) -> None:
+        """Adds a :class:`TextDisplay` to the container.
+
+        Parameters
+        ----------
+        content: :class:`str`
+            The content of the TextDisplay
+        """
+
+        text = TextDisplay(content, id=id)
+
+        return self.add_item(text)
+
+    def add_gallery(
+        self,
+        *items: Item,
+        id: int | None = None,
+    ):
+        """Adds a :class:`MediaGallery` to the container.
+
+        To append a pre-existing :class:`MediaGallery`, use the
+
+        :meth:`add_item` method instead.
+
+        Parameters
+        ----------
+        *items: :class:`MediaGalleryItem`
+            The media this gallery contains.
+        id: Optiona[:class:`int`]
+            The gallery's ID.
+        """
+        # accept raw urls?
+
+        g = MediaGallery(*items, id=id)
+
+        return self.add_item(g)
+
+    def add_file(self, url: str, spoiler: bool = False, id: int | None = None) -> None:
+        """Adds a :class:`TextDisplay` to the container.
+
+        Parameters
+        ----------
+        url: :class:`str`
+            The URL of this file's media. This must be an ``attachment://`` URL that references a :class:`~discord.File`.
+        spoiler: Optional[:class:`bool`]
+            Whether the file is a spoiler. Defaults to ``False``.
+        id: Optiona[:class:`int`]
+            The file's ID.
+        """
+
+        f = File(url, spoiler=spoiler, id=id)
+
+        return self.add_item(f)
+
+    def add_separator(
+        self,
+        *,
+        divider: bool = True,
+        spacing: SeparatorSpacingSize = SeparatorSpacingSize.small,
+        id: int | None = None,
+    ) -> None:
+        """Adds a :class:`Separator` to the container.
+
+        Parameters
+        ----------
+        divider: :class:`bool`
+            Whether the separator is a divider. Defaults to ``True``.
+        spacing: :class:`~discord.SeparatorSpacingSize`
+            The spacing size of the separator. Defaults to :attr:`~discord.SeparatorSpacingSize.small`.
+        """
+
+        s = Separator(divider=divider, spacing=spacing, id=id)
+
+        return self.add_item(s)
+
+    def copy_text(self) -> str:
+        """Returns the text of all :class:`~discord.ui.TextDisplay` items in this container. Equivalent to the `Copy Text` option on Discord clients."""

```

Comment:
```suggestion
        """Returns the text of all :class:`~discord.ui.TextDisplay` items in this container.
        Equivalent to the `Copy Text` option on Discord clients."""
```
Not sure if this is the right format